### PR TITLE
feat(natives): expand example and update natives for enhanced functio…

### PR DESCRIPTION
**Pull request by ahcenezdh**
- Addition of usage examples for the `DeleteEntity` native in Lua, JavaScript, and C#.
- Recommendation to use `DoesEntityExist` for proper entity existence validation before calling `DeleteEntity`.
- Updates the name of the following natives:
  - `_CLEAR_WEATHER_TYPE_OVERTIME_PERSIST`
  - `_GET_VEHICLE_DOOR_DESTROY_TYPE`

### CODE REVIEW

File: MISC/ClearWeatherTypeNowPersistNetwork.md [hunk 1]
```diff
@@ -1,11 +1,12 @@
 ---
 ns: MISC
+aliases: ["_CLEAR_WEATHER_TYPE_OVERTIME_PERSIST"]
```
Comment (4mmonium) [inline MISC/ClearWeatherTypeNowPersistNetwork.md:3]: Hey there, alias is supposed to be `0x0CF97F497FE7D048` i.e.

```
aliases: ["0x0CF97F497FE7D048"]
```

Not sure why the alias is named that 🤔

File: VEHICLE/GetVehicleIndividualDoorLockStatus.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+## GET_VEHICLE_INDIVIDUAL_DOOR_LOCK_STATUS
+
+```c
+// 0xCA4AC3EAAE46EC7B
+int GET_VEHICLE_INDIVIDUAL_DOOR_LOCK_STATUS(Vehicle vehicle, int doorIndex);
+```
+
+See eDoorId declared in [`SET_VEHICLE_DOOR_SHUT`](#_0x93D9BD300D7789E5)
+
+Returns vehicle door lock state previously set with [`SET_VEHICLE_INDIVIDUAL_DOORS_LOCKED`](#_0xBE70724027F85BCD)
```
Comment (4mmonium) [inline VEHICLE/GetVehicleIndividualDoorLockStatus.md:14]: This could be moved under `## Return value`.

**Review by 4mmonium — CHANGES_REQUESTED** [review]
Hey, a couple changes needed and it's good to go, check over what I commented.
Thank you for your contribution! 😊

File: MISC/ClearWeatherTypeNowPersistNetwork.md (hunk 1 above)
Comment (ahcenezdh) [reply MISC/ClearWeatherTypeNowPersistNetwork.md:3]: Should be good

File: VEHICLE/GetVehicleIndividualDoorLockStatus.md (hunk 2 above)
Comment (ahcenezdh) [reply VEHICLE/GetVehicleIndividualDoorLockStatus.md]: Should be good

**Comment by ahcenezdh** [comment]
Hey it's good!

**Comment by ahcenezdh** [comment]
Btw you can close #850 since it was the same

**Review by 4mmonium — APPROVED** [review]
Approved, thank you 🎉

**Pull request merged**